test/src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -444,6 +452,10 @@ public void after() throws Exception {
             clients.clear();
 
         } finally {
+            // Make sure the unit test flag gets reset in case a test turned
```

Comment:
BTW _reset_ in the context of a `boolean` field is normally understood to mean _set to `false`_, making this comment a little confusing.